app/renderer/components/common/contextMenu.js

```diff
@@ -49,6 +49,22 @@ class ContextMenuItem extends ImmutableComponent {
   get hasAccelerator () {
     return this.accelerator !== null
   }
+
+  getNewCoordinate (e) {
+    let node = e.target
+    while (node && node.classList && !node.classList.contains('contextMenuItem')) {
```

Comment:
not in the scope of this PR to fix it but can you add a TODO comment indicating that we shouldn't rely on classList for node checks?